doc/engineering-reference/src/loop-equipment-sizing-and-other-design-data/component-sizing.tex

```diff
@@ -716,9 +762,102 @@ \subsubsection{Rated Water Flow Rate}\label{rated-water-flow-rate-2}
 
 The calculation is identical to that done for \emph{Coil:Cooling:Water} , which is the coil design load divided by the \emph{Loop Design Temperature Difference} user input from \emph{Sizing:Plant.} For sizing the plant loop serving this coil, only one half of this flow rate is used since both the cooling and heating coil will save a flow rate but only one of these coils will operate at a time.
 
+%\subsubsection{Rated Total Heating Capacity}\label{rated-total-heating-capacity}
+
+%The rated total heating capacity is set equal to the rated total cooling capacity.
+
 \subsubsection{Rated Total Heating Capacity}\label{rated-total-heating-capacity}
 
-The rated total heating capacity is set equal to the rated total cooling capacity.
+The following calculations are performed to determine the rated total heating capacity.
```

Comment:
Hmm, are you sure it's not set equal? Looks like the code sets [`RatedCapHeatDes = DXCoolCap;` here](https://github.com/NREL/EnergyPlus/blob/b1c7a5b4634ed662ef93c23b195ba084a09df3c6/src/EnergyPlus/WaterToAirHeatPumpSimple.cc#L1894). 
Actually, all of the coefficients are already explained in the Engineering Reference section titled "Single Speed Equation-Fit Model". Maybe not real obvious, but A1-F5 are noted after the equations and then identified in the following section "Coefficient estimation procedure". So the best remedy might be to add some text in the IO Reference pointing to these sections in the Engineering Reference.

Reply:
You are right, my mistake. I got confused by the heating mode simulation code [here](https://github.com/NREL/EnergyPlus/blob/b1c7a5b4634ed662ef93c23b195ba084a09df3c6/src/EnergyPlus/WaterToAirHeatPumpSimple.cc#L2595).
Thanks, I'll make the changes in the IO reference.